cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/ObjectMappedTableProperties.java

```diff
@@ -40,22 +41,53 @@
   /**
    * The type of object in the table.
    */
-  public static final String TYPE = "type";
+  public static final String OBJECT_TYPE = "object.type";
 
   /**
    * The schema of objects in the table.
    */
-  public static final String SCHEMA = "schema";
+  public static final String OBJECT_SCHEMA = "object.schema";
+
+  /**
+   * The name of the key to use when exploring the table. Defaults to "key".
+   */
+  public static final String EXPLORE_KEY_NAME = "explore.key.name";
+
+  /**
+   * The type of the key when exploring the table. Defaults to bytes.
+   */
+  public static final String EXPLORE_KEY_TYPE = "explore.key.type";
```

Comment:
I still think this is not clear. Perhaps you can explain in the Javadocs for ObjectMappedTable: 
- when exploring an obj. mapped table, an explore schema is derived 
- every field of the object type maps to a column of the same name and corresponding type
- the row key maps to a configurable column name and type (see ObjectMappedTableProperties).

And here you say: the field name in the explore schema that represents the row key/the type of the field. 

By the way, is it possible to specify this as null, with the consequence that the row key is not present in the explore schema?

And is it possible to specify that it is the same as one of the fields of object type?